code/common/heartbeat.q

```diff
@@ -108,10 +106,9 @@ getheartbeats:{[proctype]
 
 
 \d .
-
-if[.hb.enabled;
- // set the heartbeat table to the top level namespace, to allow it to be initialised in the pub/sub routine
- @[`.;`heartbeat;:;.hb.heartbeat];
+// set the heartbeat table to the top level namespace, to allow it to be initialis    ed in the pub/sub routine
+@[`.;`heartbeat;:;.hb.heartbeat];
```

Comment:
This was originally here, but if you're in the root namespace is there any need to use `@` here?

Reply:
so should I just replace this with heartbeat:.hb.heartbeat ?

Reply:
yes sounds good